examples/distributed_search/README.md

```diff
@@ -0,0 +1,63 @@
+# Multi-Agent Copilot Search
+
+## Introduction
+
+This example application converts the user's questions into keywords to call the search engine, and then crawls a series of web pages to find answers. It involves three types of Agents, namely the UserAgent for the user, the SearcherAgent responsible for searching, and the AnswererAgent responsible for retrieve web pages.
+
+There are many web page links returned by the search engine. To improve performance, multiple instances of AnswererAgent need to run together. However, with the traditional single-process mode, even if there are multiple AnswererAgent instances, they can only obtain web page and answer questions one by one.
+
+With AgentScope's distributed mode, you can automatically make these AnswererAgent instances run at the same time to improve performance.
+
```

Comment:
1. Plz add a "prerequisite" section here to remind users to install agentscope in distributed mode ( if needed, other packages).
2. Add a note section to tell which models are already tested for this example. For other models, balabala...